store/index.ts

```diff
@@ -24,9 +38,10 @@ export interface RootState {
   };
 }
 export interface AppState extends RootState {
+  auth: AuthState;
   backdrop: BackdropState;
-  device: DeviceState;
-  displayPreferences: DisplayPreferencesState;
+  clientSettings: ClientSettingsState;
+  deviceProfile: DeviceState;
```

Comment:
Have you renamed the calls to this store? (if necessary ofc)

Reply:
Yup :)